cobalt/act.py

```diff
@@ -46,6 +51,14 @@ def __init__(self, xml=None):
     def to_xml(self):
         return etree.tostring(self.root, encoding='utf-8')
 
+    def get_namespace(self):
+        namespaces = sorted(list(AKN_NAMESPACES.values()), reverse=True)
+        possible_namespace = self.root.nsmap.get(None)
+        for ns in namespaces:
+            if ns == possible_namespace:
+                return possible_namespace
+        raise Exception(f"The XML namespace isn't (but should be) one of the following: {', '.join(namespaces.split())}")
```

Comment:
Use `ValueError` instead -- it's the appropriate error for "you gave me a bad value"